tests/fips/openssl/openssl_fips_hash.pm

```diff
@@ -40,10 +40,14 @@ sub run {
     # Remove md2 and sha, and add rmd160 and md5-sha1 from invalid hash check in fips mode
     my @invalid_hash = ("md4", "md5", "mdc2", "rmd160", "ripemd160", "whirlpool", "md5-sha1");
     for my $hash (@invalid_hash) {
-        validate_script_output "openssl dgst -$hash $tmp_file 2>&1 || true", sub { m/disabled for fips|disabled for FIPS|unknown option|Unknown digest|dgst: Unrecognized flag/ };
+        validate_script_output "openssl dgst -$hash $tmp_file 2>&1 || true", sub { m/$hash is not a known digest|unknown option|Unknown digest|dgst: Unrecognized flag/ };
```

Comment:
Ben
One quick quesion, why "d/Disabled for fips" is removed

Reply:
@rfan1 Because if we just check **disabled for fips|disabled for FIPS|** , it will miss check the bug https://bugzilla.suse.com/show_bug.cgi?id=1193859. It will still bypass the error
`routines:EVP_DigestInit_ex:disabled for FIPS:crypto/evp/digest.c:135:`
Confirm with Marcus, we would better to check the message as i.g. "md5 is not a known digest" instead of the a bit ugly message.

Reply:
If I understand Marcus correctly it's ugly but intended. https://bugzilla.suse.com/show_bug.cgi?id=1193859#c1

Reply:
@dzedro Yes, and Marcus mentioned that "perhaps the message can be adjusted to be similar to the one in sp2/sp3.", is that right? 
or do you think we need to add them back? Actually, I think it can be fixed in SP4.

Reply:
@jouyingbin Right now, it's failing everywhere :roll_eyes: 
I have seen the poo# and that it was discussed with Marcus,
but I don't understand how should the correct behavior look like.
I would love to have the previous regex, as there is bug for the issue.

Reply:
@jouyingbin Guess I now understand what behavior is expected, please review https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13935